cdap-app-fabric/src/main/java/io/cdap/cdap/config/PreferencesService.java

```diff
@@ -101,10 +115,9 @@ private void setConfig(EntityId entityId,
   /**
    * Validate the profile status is enabled and set the preferences
    */
-  private void setConfig(ProfileStore profileStore, PreferencesTable preferencesTable, EntityId entityId,
-                         Map<String, String> propertyMap)
-    throws NotFoundException, ProfileConflictException, BadRequestException, IOException {
-
+  private void setConfig(ProfileStore profileStore, PreferencesTable preferencesTable,
+                         @Name("entityId") EntityId entityId, Map<String, String> propertyMap) throws Exception {
```

Comment:
Why do we need a name annotation on this?

Reply:
Failed to clean it up. Leftover from previous change.

Reply:
Done